server/server.js

```diff
@@ -33,6 +33,7 @@ function allLangs() {
 }
 
 function prodLangs() {
+  // eslint-disable-next-line security/detect-non-literal-require
```

Comment:
Unrelated, but this was the only remaining ESLint warning.
Looking again, that could have been normalized out by potentially doing `require('../package.json')` and letting Node.js resolve paths internally (like it does elsewhere for `require('./utils')`.